belvedere/src/main/java/zendesk/belvedere/ImageStreamPresenter.java

```diff
@@ -1,85 +1,38 @@
 package zendesk.belvedere;
 
 
-import android.Manifest;
-import android.net.Uri;
-import android.os.Build;
-import android.text.TextUtils;
-
 import java.util.List;
 
 class ImageStreamPresenter implements ImageStreamMvp.Presenter {
 
     private final ImageStreamMvp.Model model;
     private final ImageStreamMvp.View view;
 
-    ImageStreamPresenter(ImageStreamMvp.Model model, ImageStreamMvp.View view) {
+    private final ImageStreamDataSource imageStreamDataSource;
+
+    ImageStreamPresenter(ImageStreamMvp.Model model, ImageStreamMvp.View view, ImageStreamDataSource imageStreamDataSource) {
```

Comment:
What's the purpose of `imageStreamDataSource` in this class? I cannot find any usages of it, do we need to pass it in a constructor and later store it in a field?